clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/StreamInvocationMessage.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+package com.microsoft.aspnet.signalr;
+
+public class StreamInvocationMessage extends HubMessage {
```

Comment:
Can this inherit from InvocationMessage instead? Just update the references to `HubMessageType.INVOCATION` with `HubMessageType.STREAM_INVOCATION`